[dpc-3630] Jd me/dpc 3630 add dpc portal (#1954)
CMSgov/dpc-app: dpc-portal/README.md

# README

This is the Data Point of Care (DPC) website. It will allow users to sign up for the DPC service, get credentialed and configure their account.

# Running via Docker

The DPC website can be run locally via docker. Follow the below steps to build and run the website into a Docker container.

- Run `make start-portals` to start dpc-portal in a docker container.

>   - This will also start dpc-web and dpc-admin, which are no longer used in production.

Reviewer: Would suggest rewording this a bit since these apps were never used in production 👋 

```suggestion
  - This will also start dpc-web and dpc-admin, which are only used in the sandbox environment.
```
